File: rafael/variaveis/TestaFormatacao.cs
class TestaFormatacao 
{
	static void Main() 
	{
		string nome = "Jonas Hirata";
		int idade = 30;
		double peso = 49.7345;

		System.Console.WriteLine("O {0} tem {1} anos e pesa {2:F2} kg", nome, idade, peso);
	}
}

File: rafael/variaveis/TestaFormatacao.java
class TestaFormatacao {
	public static void main(String[] args) {
		String nome = "Jonas Hirata";
		int idade = 30;
		double peso = 49.7345;

		System.out.printf("O %1$s tem %2$d anos e pesa %3$.2f kg\n", nome, idade, peso);
	}
}
